blockchain/README.md

```diff
@@ -1,7 +1,44 @@
 # Blockchain
 
-A Blockchain is a type of distributed ledger technology (DLT) that consists of growing list of records, called blocks, that are securely linked together using cryptography.
+A Blockchain is a type of distributed ledger technology **(DLT)** that consists of growing list of records, called **blocks**, that are securely linked together using **cryptography**.
```

Comment:
```suggestion
A Blockchain is a type of **distributed ledger** technology (DLT) that consists of growing list of records, called **blocks**, that are securely linked together using **cryptography**.
```

Reply:
Thank you for the feedback. I have made the requested changes in this file. Please revert back if any other changes are required in the document.